smart-contracts/zos.json

```diff
@@ -1,5 +1,5 @@
 {
-  "zosversion": "2",
+  "zosversion": "2.2",
```

Comment:
We may need the same update in the other zos.*.json files

Reply:
yes. I can do that in the PR I have going right now.